app/views/merchant/items/index.html.erb

```diff
@@ -3,6 +3,19 @@
 <h4>Items</h4>
 <section class="items">
   <% @merchant.items.each do |item| %>
-    <li><%= link_to item.name, "/merchants/#{@merchant.id}/items/#{item.id}" %></li>
```

Comment:
link to item.name? is this like saying link_to that items show page?

Reply:
Or is it just outputting the item name as the link?

Reply:
first argument (item.name) is the actualy text that the link will output and the second argument is the path